tools/Common.NoTargets.props

```diff
@@ -2,7 +2,6 @@
 <Project>
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0</TargetFrameworks>

```

Comment:
This should be kept because it fairly significantly improves build perf by only delegating to one TFM. Other than that, it doesn't matter what TFM it is here.